api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantLifecycleIntegrationTests.java

```diff
@@ -323,6 +356,19 @@ private LocalDate progressThroughPaymentCyclesForPregnancy(LocalDate expectedDel
         return expectedDeliveryDate.minusDays((long) CYCLE_DURATION * numCycles);
     }
 
+    private LocalDate progressThroughPaymentCyclesForPregnancyWithEligibilityOverride(LocalDate expectedDeliveryDate, UUID claimId, int numCycles)
+            throws NotificationClientException, JsonProcessingException {
+        for (int i = 0; i < numCycles; i++) {
+            resetNotificationClient();
+            repositoryMediator.ageDatabaseEntities(CYCLE_DURATION);
+            wiremockManager.stubIneligibleEligibilityResponse();
```

Comment:
I would remove this mock as the eligibility service is not called.

Reply:
done